.gitignore

```diff
@@ -201,4 +201,7 @@ tools/MapAtmosFixer/MapAtmosFixer/bin/*
 .atom-build.json
 
 #Jukebox audio files
-/config/jukebox_music/**/*
\ No newline at end of file
+/config/jukebox_music/**/*
+*.dmm
+_maps/nanostation.dm
+_maps/nanotstation.json
```

Comment:
??????????????????????????????????

Reply:
That was just me telling get not to merge my current WIP map.